sql/step_04_all_following_steps.sql

```diff
@@ -87,3 +87,4 @@ source step_90_login_retries.sql;
 source step_91_alter_laboratory_with_status.sql;
 source step_92_add_dicom_data_table.sql;
 source step_93_patient_consensus_table.sql;
+source step_94_missing_user_related_fk.sql;
```

Comment:
FWIW, both this PR and https://github.com/informatici/openhospital-core/pull/1002 use `step_94`.   One of them is going to have to change 😏

Reply:
Renamed file to step_95 with https://github.com/informatici/openhospital-core/pull/998/commits/455ca67ce1a5e874188203ebe2ae395ac0db9914